spec/requests/v1/pacbio/runs_spec.rb

```diff
@@ -1411,19 +1437,20 @@
         it 'creates the correct number of used_aliquots' do
           expect do
             patch v1_pacbio_run_path(run), params: body, headers: json_api_headers
-          end.to change(Aliquot, :count).by(-7)
+          end.not_to change(Aliquot, :count)
```

Comment:
If we are removing aliquots shouldn't this be changed? Or are we adding the same amount we are removing?

Reply:
Yes, we are adding 1 and removing 1.